ui/app/src/modules/Content/Authoring/NewContentDialog.tsx

```diff
@@ -158,58 +121,32 @@ export interface NewContentDialogStateProps extends NewContentDialogBaseProps {
   onDismiss?: StandardAction;
 }
 
-function ContentTypesGrid(props: ContentTypesGridProps) {
-  const { resource, isCompact, onTypeOpen, getPrevImg, selectedContentType } = props;
-  const classes = useStyles({});
-  const filterContentTypes = resource.read();
-  return (
-    <Grid container spacing={3} className={classes.cardsContainer}>
-      {filterContentTypes.map((content) => (
-        <Grid item key={content.label} xs={12} sm={!isCompact ? 4 : 6}>
-          <NewContentCard
-            isCompact={isCompact}
-            headerTitle={content.label}
-            subheader={content.form}
-            img={getPrevImg(content)}
-            onClick={onTypeOpen(content)}
-            isSelected={content.name === selectedContentType}
-          />
-        </Grid>
-      ))}
-    </Grid>
-  );
-}
-
 export default function NewContentDialog(props: NewContentDialogProps) {
   return (
     <Dialog open={props.open} onClose={props.onClose} fullWidth maxWidth="md">
-      <NewContentDialogBody {...props} />
+      <NewContentDialogWrapper {...props} />
     </Dialog>
   );
 }
 
-function NewContentDialogBody(props: NewContentDialogProps) {
```

Comment:
Looks like the merge resulted in some changes getting lost. These renames should persist. ...DialogBody is the correct name.